pytorch_lightning/accelerators/tpu.py

```diff
@@ -17,3 +25,27 @@ def setup(self, trainer, model):
         if not isinstance(self.training_type_plugin, (SingleTPUPlugin, TPUSpawnPlugin)):
             raise MisconfigurationException("TPUs only support a single tpu core or tpu spawn training.")
         return super().setup(trainer, model)
+
+    def optimizer_step(
```

Comment:
Suggestion: I would prefer to have only run_optimizer_step, so hooks live in the parent.